ntbs-service/appsettings.Development.json

```diff
@@ -6,6 +6,7 @@
   "AdfsOptions": {
     "Wtrealm": "https://localhost:5001/",
     "DevGroup": "Global.NIS.NTBS.NTA",
+    "AdminUserGroup": "Global.NIS.NTBS.ADMIN",
```

Comment:
You'll need to set it in `appsettings.json`, since you're using it as a app config value, not just as dev auth help!